metricbeat-tests/assertions.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 // assertHitsArePresent returns an error if no hits are present
+//nolint:unused
```

Comment:
Umm, these methods are used, why is this needed?

Reply:
I believe they are false positives in `golangci-lint`: https://apm-ci.elastic.co/blue/organizations/jenkins/beats%2Fpoc-metricbeat-tests-poc-mbp/detail/PR-67/1/pipeline